### PR TITLE
Added plugin pwmbuzzer

### PR DESCRIPTION
**Pull request by stealthmonkey99**
## Description
This PR adds my first OctoPrint plugin `pwmbuzzer` to the repo to register it.  See full plugin details in pwmbuzzer.md.

## Test Plan
- [x] verified my listing displays appropriately with jekyll
- [x] verified plugin functionality on local instance
- [x] verified plugin functionality on Raspberry Pi 4B with a passive buzzer attached to GPIO16

### CODE REVIEW

**Comment by jneilliii** [comment]
nice plugin idea, kinda of like an extension to my M300 Player plugin. will be reviewing soon.

File: _plugins/pwmbuzzer.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+description: Uses PWM via GPIO to sound tones whenever an M300 command is encountered
+authors:
+- Matt Bielich
+license: AGPLv3
+
+date: 2022-03-29
+
+homepage: https://github.com/stealthmonkey99/OctoPrint-PWMBuzzer
+source: https://github.com/stealthmonkey99/OctoPrint-PWMBuzzer
+archive: https://github.com/stealthmonkey99/OctoPrint-PWMBuzzer/archive/master.zip
```
Comment (jneilliii) [inline _plugins/pwmbuzzer.md:15]: I know GitHub will redirect this properly, but should be main.zip, not master.zip.

**Review by jneilliii — CHANGES_REQUESTED** [review]
just a minor change needed. really nice plugin and the composer is super cool, great job.

File: _plugins/pwmbuzzer.md (hunk 1 above)
Comment (stealthmonkey99) [reply _plugins/pwmbuzzer.md]: oh good eye, thanks!  And thank you for the review - I was inspired by your plugin a while back and finally got around to hooking up a buzzer 😆

**Review by jneilliii — APPROVED** [review]

**Pull request merged**